packages/microcosm/src/observable.js

```diff
@@ -94,9 +96,13 @@ export class Subscription implements Unsubscribable {
       this._cleanup = subscriber.call(origin, subscriptionObserver) || noop
     } catch (error) {
       subscriptionObserver.error(error)
+
+      if (observer.error === noop) {
+        scheduler().raise(error)
+      }
```

Comment:
New: If an exception is raised in a subscriber and there isn't an error handler, we raise with the scheduler. This happens for like:

```
let obs = new Observable(observer => {
  throw "Crud"
})

obs.subscribe(next => {}) // Raises an exception with the scheduler
obs.subscribe(next => {}, error => {}) // Doesn't

Reply:
worth adding this as a comment in the code?

Reply:
Certainly.